content/release/1.4.1.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Release 1.4.1 available
+date: 2024-11-24
+linked: true
+---
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+![Hot-Springs-haiku](releases/1.4.0.jpg)
+
```

Comment:
I guess we can omit the same image.

```suggestion
```

Reply:
Theoretically, this nature park is also a park tag of ozone-1.4.1, which is an independent release version, so should we remove this?

Reply:
I think so, but you can wait for other opinions before updating the patch.

Reply:
As I said in the new website proposal I don't like the whole National Park Haiku thing 😄 I'm totally ok with dropping it here.